ci/test.sh

```diff
@@ -18,7 +18,7 @@ export JENKINS_USER=${JENKINS_USER_NAME}
 export GRADLE_ENTERPRISE_ACCESS_KEY=${DEVELOCITY_ACCESS_KEY}
 
 # Execute maven test
-MAVEN_OPTS="-Duser.name=${JENKINS_USER} -Duser.home=/tmp/jenkins-home" ./mvnw -s settings.xml clean test -P${PROFILE} -DrunLongTests=${LONG_TESTS:-false} -U -B
+MAVEN_OPTS="-Duser.name=${JENKINS_USER} -Duser.home=/tmp/jenkins-home" ./mvnw -s settings.xml clean test -P${PROFILE} -DrunLongTests=${LONG_TESTS:-false} -Dgradle.cache.local.enabled=false -Dgradle.cache.remote.enabled=false -U -B
```

Comment:
@erichaagdev Can you provide guidance on how to not skip tests when running builds against different build stages where the only change is the Redis or JDK version?

Reply:
This PR will cause all tests to re-run in builds with these Maven options since it disables the build cache. 

Test results come from cache when all inputs to the surefire/failsafe goal match a previous invocation in the cache. These inputs include the JDK version and the runtime classpath. So, assuming you have Redis in the classpath, these tests should re-execute on these sorts of changes even with the cache enabled.

Have you seen different behavior than this?

Reply:
> This PR will cause all tests to re-run in builds with these Maven options since it disables the build cache.

That is intended because we want to run integration tests against a different Redis version. Our Redis service is an externally managed service that does not affect the classpath. We would need to have to tell the caching inputs about a changed cache key somehow.

Reply:
I see. Essentially, you want to represent the version of Redis as an input to the surefire/failsafe goals. The documentation for that can be found [here](https://docs.gradle.com/enterprise/maven-extension/#adding_inputs_and_outputs).

More specifically, you could either:
* Pass in the Redis version as a system property to `<argLine>`
* Specify some file in the Redis installation that represents the version as an input file
Either of these would invalidate the cache when the Redis version changes.

Reply:
Thanks for the detail. `<argLine>` seems the most promising approach for now.